statement-table/src/generic.rs

```diff
@@ -57,8 +57,8 @@ pub trait Context {
 	/// Members are meant to submit candidates and vote on validity.
 	fn is_member_of(&self, authority: &Self::AuthorityId, group: &Self::GroupId) -> bool;
 
-	/// requisite number of votes for validity from a group.
-	fn requisite_votes(&self, group: &Self::GroupId) -> usize;
+	/// Get a validator group size.
+	fn get_group_size(&self, group: &Self::GroupId) -> Option<usize>;
```

Comment:
I don't think it should ever return `None` under the current usage model but it could if there's a bug.

Still, the method needs to return Option because it queries a `Hashmap` and we can't know at compile-time whether or not the validator group is present for all the paraIDs.